### PR TITLE
Adding hootsuite

### PR DESCRIPTION
**Pull request by DianaDascalu2**

### CODE REVIEW

File: src/expressvpn/index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h3><i>Scraper-UI.</i></h3>
                             <img alt="expressvpn"  src="https://upload.wikimedia.org/wikipedia/en/thumb/7/79/ExpressVPN-logo.svg/261px-ExpressVPN-logo.svg.png?20210118095637" />
                         </div>
                         <div class="box2">
-                            <a href="#" target="_blank">
+                            <a href="https://www.expressvpn.com/jobs/job-openings" target="_blank">
                             Careers
                             </a>
                             <a href="#" target="_blank">
```
Comment (lalalaurentiu) [inline src/expressvpn/index.html:47]: you forgot targetare.ro link
```suggestion
                            <a href="targetare.ro_link" target="_blank">
```

File: src/inthepocket/index.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                <div class="flip-card-container">
+                    <div class="flip-card">
+                        <div class="box">
+                            <img alt="inthepocket"  src="https://media.licdn.com/dms/image/C4E0BAQEEf_0PuRSBQg/company-logo_200_200/0/1636019709603/in_the_pocket_logo?e=1708560000&v=beta&t=Pq7AD1__VKnMwad2fPXEXpdcq3DEHGsmXCIt-We495A" />
+                        </div>
+                        <div class="box2">
+                            <a href="https://www.inthepocket.com/careers" target="_blank">
+                            Careers
+                            </a>
+                            <a href="#" target="_blank">
```
Comment (lalalaurentiu) [inline src/inthepocket/index.html:47]: you forgot targetare.ro link
```suggestion
                            <a href="targetare.ro_link" target="_blank">
```

File: src/qiagen/index.html [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+                <div class="flip-card-container">
+                    <div class="flip-card">
+                        <div class="box">
+                            <img alt="qiagen"  src="https://www.qiagen.com/sfc/images/qiagen-logo.png" />
+                        </div>
+                        <div class="box2">
+                            <a href="https://www.qiagen.com/us/careers/jobs/index?searchCriteria[0][key]=LOV26&searchCriteria[0][values][]=16537&searchCriteria[1][key]=Resultsperpage&searchCriteria[1][values][]=50" target="_blank">
+                            Careers
+                            </a>
+                            <a href="#" target="_blank">
```
Comment (lalalaurentiu) [inline src/qiagen/index.html:47]: you forgot targetare.ro like
```suggestion
                            <a href="targetare.ro_link" target="_blank">
```

File: src/sp/index.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+            <p class="close">X</p>
+            <div class="progress active"></div>
+        </div>
+        <!-- End Section Alert -->
+        <section class="company">
+            <div class="company-container">
+                <div class="flip-card-container">
+                    <div class="flip-card">
+                        <div class="box">
+                            <img alt="sp"  src="None" />
```
Comment (lalalaurentiu) [inline src/sp/index.html:41]: you forgot image url
```suggestion
                            <img alt="sp"  src="image_url" />
```

File: src/sp/index.html [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+                        </div>
+                        <div class="box2">
+                            <a href="https://careers.spglobal.com/jobs?locations=Bucharest,Bucure%C5%9Fti,Romania&page=1" target="_blank">
+                            Careers
+                            </a>
+                            <a href="#" target="_blank">
```
Comment (lalalaurentiu) [inline src/sp/index.html:47]: you forgot targetare.ro link
```suggestion
                            <a href="targetare.ro_link" target="_blank">
```

**Review by lalalaurentiu — CHANGES_REQUESTED** [review]

File: src/expressvpn/index.html (hunk 1 above)
Comment (DianaDascalu2) [reply src/expressvpn/index.html:47]: Got it.

**Comment by lalalaurentiu** [comment]
@DianaDascalu2 You didn't have to accept my suggestions. In the 'href' attribute, you should search for links related to careers, targetare.ro, and add the image link.

**Pull request closed without merging**